src/openforms/logging/logevent.py

```diff
@@ -439,20 +447,97 @@ def submission_export_list(form: "Form", user: "User"):
+            logic_data,
+            keys_mapped_to_component,
+            keys_mapped_to_form,
+            resulting_data,
+        )
+
+        stringified_json_tree = stringify_json_tree(source_components)
+        print(stringified_json_tree)
+
+        # will remove duplicate components that are involved in several rules
+        remove_duplicate_components_from_json_tree(
+            source_components, involved_components
+        )
+        targeted_components = []
+        for action in rule.actions:
+            log_action = {}
+
+            # Action verifications
+            action_details = action["action"]
+            log_action["value"] = ""
+            log_action["type"] = action_details["type"]
+            if action_details["type"] == LogicActionTypes.value:
+                log_action["value"] = action_details["value"]
+                log_action["key"] = action["component"]
+            elif action_details["type"] == LogicActionTypes.property:
+                log_action["value"] = action_details["property"]["value"]
+                log_action["state"] = action_details["state"]
+                log_action["key"] = action["component"]
+            elif action_details["type"] == LogicActionTypes.variable:
+                log_action["key"] = action["variable"]
+                logic = JsonLogicTest.from_expression(action_details["value"])
+                src_components = convert_json_logic_in_json_tree(
+                    logic,
+                    keys_mapped_to_component,
+                    keys_mapped_to_form,
+                    resulting_data,
+                )
+                log_action["value"] = stringify_json_tree(src_components)
+                log_action["isVariable"] = True
+            targeted_components.append(log_action)
+
+        # Assuming key is unique in all form steps
+        for targeted_component in targeted_components:
+            key = targeted_component.get("key")
+            # Adding the label of the key and step name associated if key exists
+            if not key:
+                continue
+            # Might happens that a form variable is not related to a component
+            if targeted_component.get(
+                "isVariable"
+            ) and not keys_mapped_to_component.get(key):
+                variable = submission.form.formvariable_set.get(key=key)
+                targeted_component["label"] = variable.name
+            else:
+                targeted_component["label"] = keys_mapped_to_component[key].get(
+                    "label", ""
+                )
+                targeted_component["step_name"] = keys_mapped_to_form[
+                    key
+                ].form_definition.name
+            break
 
         evaluated_rule_data = {
-            "source_components": rule.json_logic_trigger,
-            "targeted_components": rule.actions,
+            "rule_as_str": stringified_json_tree,
```

Comment:
```suggestion
            "readable_rule": stringified_json_tree,
```
perhaps?

I think you'd still want to include the raw underlying expression as well, just for logging purposes. It's possible that the logic rule is edited after it's logged, and then all you have left is the string-representation. If you have to debug things based on logs, it would definitely be useful to have the raw logic expression stored. This also applies in the event of bugs in the string-representation calculation - we would be able to generate a new string representation and fix that.